docs/src/pages/discover-more/related-projects/related-projects.md

```diff
@@ -12,40 +12,13 @@ Feel free to submit a pull request to add more projects; we will accept them if
 
 ## Material-UI Specific Projects
```

Comment:
```suggestion
## Libraries
```
?

No need to differentiate between MUI specific, and general but complementary when the latter have been deleted.

Reply:
👌